commcare_connect/opportunity/models.py

```diff
@@ -599,3 +600,14 @@ class UserInvite(models.Model):
     opportunity_access = models.OneToOneField(OpportunityAccess, on_delete=models.CASCADE, null=True, blank=True)
     message_sid = models.CharField(max_length=50, null=True, blank=True)
     status = models.CharField(max_length=50, choices=UserInviteStatus.choices, default=UserInviteStatus.invited)
+
+
+class CatchmentArea(models.Model):
+    opportunity = models.ForeignKey(Opportunity, on_delete=models.CASCADE)
+    latitude = models.DecimalField(max_digits=11, decimal_places=8)
+    longitude = models.DecimalField(max_digits=11, decimal_places=8)
+    radius = models.IntegerField(default=1000)
+    opportunity_access = models.ForeignKey(OpportunityAccess, null=True, on_delete=models.DO_NOTHING)
+    active = models.BooleanField(default=True)
+    name = models.CharField(max_length=255)
+    site_code = models.SlugField(max_length=255)
```

Comment:
This can have a DB constraint to ensure it is unique to an opportunity

Reply:
@calellowitz  Done